Resources/Prototypes/Recipes/Lathes/misc.yml

```
@@ -237,3 +237,11 @@
   completetime: 3
   materials:
     Plastic: 200
+
+- type: latheRecipe
+  id: TankHarness
```

```suggestion
  id: ClothingOuterVestTank
```
name it the same please to keep the naming convention